### PR TITLE
Implements Linux platform and RH8 compiler.

### PR DESCRIPTION
**Pull request by OmniBlade**
Another x86 related one, apart from bugs in x86 diffing I think there shouldn't really be any other concerns with this one.

### CODE REVIEW

File: frontend/src/app/(navfooter)/credits/page.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ const ICON_SOURCES = {
     "Tiger-like-x by Althepal": "https://commons.wikimedia.org/wiki/File:Tiger-like-x.svg",
     "Saturn by JustDanPatrick": "https://upload.wikimedia.org/wikipedia/commons/archive/7/78/20220518145749%21Sega_Saturn_Black_Logo.svg",
     "MS-DOS by Microsoft": "https://commons.wikimedia.org/wiki/File:Msdos-icon.svg",
+    "Tux by Larry Ewing": "https://commons.wikimedia.org/wiki/File:Tux.svg",
```
Comment (ethteck) [inline frontend/src/app/(navfooter)/credits/page.tsx:29]: it appears there are 3 authors actually - probably best to list all of them. (Simon Budig and Garrett LeSage too)

**Review by nanaian — APPROVED** [review]

**Comment by ethteck** [comment]
Holding off on this one until we get https://github.com/simonlindholm/asm-differ/issues/123

**Comment by ethteck** [comment]
Also it's supposedly only going to be viable for one game. so unless omniblade has multiple people working on said project, we might want to pass on this one actually. discussion: https://discord.com/channels/897066363951128586/897066363951128590/1135515104788619357

**Comment by mkst** [comment]
Closing this, we can always salvage the bits of it if we want to add Linux as a platform in future.

**Pull request closed without merging**